### DISCUSSION TITLE
Update shadow when scaling model

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#4003] [Q&A] [answered]
Hi guys, I update the models size so that I have 3 sizes (S, M and L), the issue I have at the moment is that when I scale up the model the shadow doesn't update as expected. These are the lines of code I use when I resize the model. 
` modelViewer[sceneSym]._model.scale.x = 1.48387;
 
 modelViewer[sceneSym]._model.scale.y = 1.15;
 
 modelViewer[sceneSym]._model.scale.z = 1.48387;
 
 modelViewer.updateFraming();
 
 modelViewer[sceneSym].updateShadow();
`
![image_2022_12_12T15_27_00_624Z](https://user-images.githubusercontent.com/92090684/207086224-10db265e-9d33-498b-8f1b-c496fe379b70.png)
![image_2022_12_12T15_27_22_485Z](https://user-images.githubusercontent.com/92090684/207086240-ae6b584e-0482-42a0-9c98-5b3feb4488e1.png)

**Comment by elalish** [comment, marked as answer]
Try our public API instead: `modelViewer.scale = "1.48 1.15 1.48";`

**Reply by danilokrulj** [reply]
Thank you @elalish ! You are as helpful as always! That did the trick.
